app/org/maproulette/framework/mixins/SearchParametersMixin.scala

```diff
@@ -234,25 +414,29 @@ trait SearchParametersMixin {
             ),
             if (invert) AND() else OR()
           )
-        CustomParameter(s"(${query.sql()})")
-      case Some(statuses) if statuses.isEmpty => CustomParameter("")
-      case _                                  => CustomParameter("")
+        FilterGroup(List(CustomParameter(s"(${query.sql()})")))
```

Comment:
This kind of seems like a pointless wrapping. The "query" object is actually a FilterGroup, so you could just technically return the object directly.

Reply:
The point of this wrapping is to apply () around the query. Will FilterGroup do that automatically?